### PR TITLE
feat: add collaterals

### PR DESCRIPTION
**Pull request by tyler-tsai**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/helio-money exports TVL: 
        
 
 
    bsc                       163.10 M
    
    total                    163.10 M

**Comment by llamatester** [comment]
The adapter at projects/helio-money exports TVL: 
        
 
 
    bsc                       171.80 M
    
    total                    171.80 M

**Comment by g1nt0ki** [comment]
hi @tyler-tsai it is bit confusing, is this code ready for review or WIP?

**Comment by tyler-tsai** [comment]
> hi @tyler-tsai it is bit confusing, is this code ready for review or WIP?

it still work in progress, I will remove the label today.
sorry for inconvenience, do you have other convention to express WIP?

**Comment by tyler-tsai** [comment]
BTW, can I update the file name?
we rename from helio to lista dao

**Comment by llamatester** [comment]
The adapter at projects/helio-money exports TVL: 
        
 
 
    bsc                       171.63 M
    
    total                    171.63 M

**Comment by llamatester** [comment]
The adapter at projects/lista-dao exports TVL: 
        
 
 
    bsc                       171.63 M
    
    total                    171.63 M

**Comment by tyler-tsai** [comment]
@g1nt0ki 
it's all set. I rename the file from helio-money to lista-dao. It might need to re-mapping by our side. much appreciated.

File: projects/helper/coreAssets.json [hunk 1]
```diff
@@ -128,7 +128,10 @@
     "IVN": "0x6a46d878401f46b4c7f665f065e0667580e031ec",
     "ankrBNB": "0xe85afccdafbe7f2b096f268e31cce3da8da2990a",
     "aBNBb": "0xbb1aa6e59e5163d8722a122cd66eba614b59df0d",
-    "TUSD": "0x40af3827f39d0eacbf4a168f8d4ee67c121d11c9"
+    "TUSD": "0x40af3827f39d0eacbf4a168f8d4ee67c121d11c9",
+    "wstETH": "0x26c5e01524d2E6280A48F2c50fF6De7e52E9611C",
+    "BBTC": "0xF5e11df1ebCf78b6b6D26E04FF19cD786a1e81dC",
```
Comment (g1nt0ki) [inline projects/helper/coreAssets.json:133]: also, can you remove BBTC & solvBTC from here and declare it directly in your file? the tokens here are sort of whitelisted and used for pricing other tokens

**Comment by llamatester** [comment]
The adapter at projects/helio-money exports TVL: 
        
 
 
    bsc                       171.56 M
    
    total                    171.56 M

**Comment by llamatester** [comment]
Error while running adapter at projects/helio-money: 
        
 
  
    
    
    SyntaxError: /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/coreAssets.json: Unexpected token } in JSON at position 7049
    Truncated error stack:
    
        at parse (<anonymous>)
        at Object.<anonymous> (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/utils.js:1:19)
    Error saving cache: Error: ENOENT: no such file or directory, open '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
        at Object.openSync (node:fs:596:3)
        at Object.writeFileSync (node:fs:2322:35)
        at process.saveCache (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/cache.js:152:17)
        at process.emit (node:events:529:35)
        at process.exit (node:internal/process/per_thread:191:15)
        at process.handleError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/utils/handleError.js:18:11)
        at process.emit (node:events:517:28)
        at process._fatalException (node:internal/process/execution:169:25) {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
    }

**Comment by llamatester** [comment]
The adapter at projects/helio-money exports TVL: 
        
 
 
    bsc                       171.56 M
    
    total                    171.56 M

**Comment by tyler-tsai** [comment]
> hi @tyler-tsai can you revert folder name change, we can take care of project metadata in a different location, this change will break our server.

sure thing!

**Pull request merged**